core/templates/services/loader.service.ts

```diff
@@ -34,20 +32,20 @@ export class LoaderService {
    * In order to keep the variables same, static is used until migration is
    * complete.
    */
-  static loadingMessageSubject: Subject<string> = new Subject<string>();
+  static loadingMessageSubject = new EventEmitter<string>();
```

Comment:
Ditto, not sure you want to use "Subject".

Reply:
changed it to loadingMessageChangedEvent

Reply:
Ditto.

Reply:
changed it to loadingMessageChangedEventEmitter